diff --git a/requirements.txt b/requirements.txt
index f984b811..dac1b385 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -4,5 +4,4 @@ markdown
 mutagen>=1.37
 pyobjc-core; sys_platform == 'darwin'
 pyobjc-framework-Cocoa; sys_platform == 'darwin'
-PyQt5>=5.7.1
 pywin32; sys_platform == 'win32'
\ No newline at end of file
diff --git a/setup.py b/setup.py
index d26263a6..09243e71 100644
--- a/setup.py
+++ b/setup.py
@@ -772,7 +772,7 @@ args = {
         'patch_version': picard_patch_version,
     },
     'scripts': ['scripts/' + PACKAGE_NAME],
-    'install_requires': ['PyQt5', 'mutagen', 'python-dateutil'],
+    'install_requires': ['mutagen', 'python-dateutil'],
     'python_requires': '~=3.5',
     'classifiers': [
         'License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)',
