diff --git a/picard/ui/statusindicator.py b/picard/ui/statusindicator.py
index 882e39e9..3318c75d 100644
--- a/picard/ui/statusindicator.py
+++ b/picard/ui/statusindicator.py
@@ -110,8 +110,6 @@ elif not (IS_MACOS or IS_HAIKU):
 
     else:
 
-        from picard import PICARD_DESKTOP_NAME
-
         DBUS_INTERFACE = 'com.canonical.Unity.LauncherEntry'
 
         class UnityLauncherEntryService(QObject):
@@ -177,7 +175,7 @@ elif not (IS_MACOS or IS_HAIKU):
             def __init__(self, window):
                 super().__init__()
                 bus = QDBusConnection.sessionBus()
-                self._service = UnityLauncherEntryService(bus, PICARD_DESKTOP_NAME)
+                self._service = UnityLauncherEntryService(bus, 'picard_picard.desktop')
 
             @property
             def is_available(self):
