diff --git a/picard/ui/statusindicator/unity.py b/picard/ui/statusindicator/unity.py
index 64ab8e388..7fbe6779b 100644
--- a/picard/ui/statusindicator/unity.py
+++ b/picard/ui/statusindicator/unity.py
@@ -31,8 +31,6 @@ from PyQt6.QtDBus import (
     QDBusMessage,
 )
 
-from picard import PICARD_DESKTOP_NAME
-
 from . import AbstractProgressStatusIndicator
 
 
@@ -104,7 +102,7 @@ class UnityLauncherEntryStatusIndicator(AbstractProgressStatusIndicator):
     def __init__(self, window):
         super().__init__()
         bus = QDBusConnection.sessionBus()
-        self._service = UnityLauncherEntryService(bus, PICARD_DESKTOP_NAME)
+        self._service = UnityLauncherEntryService(bus, 'picard_picard.desktop')
 
     @property
     def is_available(self):
